Sources/swift-format/PrintVersion.swift

```diff
@@ -12,5 +12,5 @@
 
 func printVersionInformation() {
   // TODO: Automate updates to this somehow.
-  print("main")
+  print("6.2.0")
```

Comment:
Wouldn't it be more appropriate if it were in the form of `"602.0.0"`?

Reply:
Yeah, I agree. That was my "which also means the check I added here doesn't work" since publishing the release is for the library and not executable. I'd like to do something here to avoid forgetting to update this, but not sure what that something is right now.